aw_watcher_window/main.py

```diff
@@ -15,12 +15,6 @@
 
 logger = logging.getLogger(__name__)
 
-# run with LOG_LEVEL=DEBUG
-log_level = os.environ.get("LOG_LEVEL")
-if log_level:
-    logger.setLevel(logging.__getattribute__(log_level.upper()))
-
-
```

Comment:
Why was this removed?

Reply:
This is now in aw-client https://github.com/ActivityWatch/aw-client/commit/4b00b98bb39114aedb819e113a7e85f05027024b